templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,250 @@
-    <div class="u-fixed-width">
-      <h2>Kubernetes vs Docker: CRI-O and OCI</h2>
-      <p>
-        With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. '<a href="/kubernetes/what-is-kubernetes#container-orchestration">Container orchestration</a>'' becomes a problem space of its own. The rise of <a href="/kubernetes">Kubernetes</a> is the industry's answer to the container orchestration problem, providing a way to manage containers at scale efficiently.
-      </p>
-      <p>
-        CRI (Container Runtime Interface) is a Kubernetes API to interface between Kubernetes orchestration features and container runtimes, such as containerd or CRI-O. The <a href="https://opencontainers.org/">Open Container Initiative</a> (OCI) defines a set of specifications to enable all these technologies to work together. Docker, runc, containerd, CRI-O are all OCI compatible.
-      </p>
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h2>Docker for beginners: docker-cli, containerd, and runc</h2>
+      </div>
+      <div class="col">
+        <p>
+          Docker is many things: a community, a company, and multiple software projects. The so-called “Docker developer experience” uses three projects: <a href="https://github.com/docker/cli">docker-cli</a>, <a href="https://containerd.io/">containerd</a>, and <a href="https://github.com/opencontainers/runc">runc</a>. The Docker command-line mostly interfaces with the containerd daemon to provide a great end-user experience. Containerd manages and supervises containers, images, storage, and networking. Finally, runc is the container runtime. It consists of all the code used to interact with container system features (think user namespaces, cgroups, and so on).
+        </p>
+      </div>
     </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-7">
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h2>Kubernetes vs. Docker: CRI-O and OCI</h2>
+      </div>
+      <div class="col">
         <p>
-          Not clear on how Kubernetes vs Docker differ?
-          <br />
-          <a href="/blog/kubernetes-versus-docker">Read the Kubernetes vs Docker blog&nbsp;&rsaquo;</a>
+          With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. <a href="https://www.ubuntu.com/kubernetes/what-is-kubernetes#container-orchestration">"Container orchestration″</a> becomes a problem space of its own. The rise of <a href="https://ubuntu.com/kubernetes">Kubernetes</a> is the industry’s answer to the container orchestration problem, providing a way to manage containers at scale efficiently.
```

Comment:
```suggestion
          With more and more applications deployed on containers, simple orchestration tools are not enough to support complex production environments. <a href="/kubernetes/what-is-kubernetes#container-orchestration">"Container orchestration″</a> becomes a problem space of its own. The rise of <a href="/kubernetes">Kubernetes</a> is the industry’s answer to the container orchestration problem, providing a way to manage containers at scale efficiently.
```
Since these are internal links we don't need to provide the full path